tests/operators/test_python.py

```diff
@@ -272,6 +272,18 @@ def func(dag):
         with pytest.raises(ValueError, match=error_message):
             ti.run()
 
+    def test_unpack_operator_kwargs(self):
+        """Makes sure that keyword arguments are not continually processed after an unpack operator."""
+
+        error_message = "Should be run"
+
+        def func(*unpacked_args, keyword_only_arg):
+            raise RuntimeError(error_message)
```

Comment:
This could be a pass, but it doesn't actually ensure the method is actually run.